testIntegration/tests/userAgent.test.ts

```diff
@@ -8,66 +8,81 @@
 import fs from "fs-extra";
 import nock from "nock";
 import { expect } from "chai";
-import { ShopperCustomers } from "commerce-sdk/dist/customer/customer";
+import { ShopperLogin, ShopperLoginTypes } from "commerce-sdk/dist";
 
 // Helper to make the user agent being tested a bit more obvious
 async function mockRequestWithUserAgent(userAgent: string): Promise<void> {
   nock("http://somewhere")
+    .get("/organizations/foo/oauth2/authorize")
     .matchHeader("user-agent", userAgent)
-    .post("/organizations/foo/customers/actions/login")
+    .query(true) // this is required for nock to ignore other headers in the request
     .reply(200, {}, { Authorization: "Bearer AUTH_TOKEN" });
 }
 
+const params = {
+  organizationId: "foo",
+  redirect_uri: "redirect_uri",
+  response_type: "code" as const,
+  client_id: "client_id",
+  scope: ShopperLogin.AuthorizeCustomerScopeEnum.Openid,
+  state: "state",
+  usid: "usid",
+  hint: "hint",
+  channel_id: "channel_id",
+  code_challenge: "code_challenge",
+};
+
 describe("Custom user agent header", () => {
   let sdkUserAgent: string;
   before(async () => {
     const { version } = await fs.readJson(
       require.resolve("commerce-sdk/package.json")
     );
-    sdkUserAgent = `commerce-sdk@${version};`;
+    sdkUserAgent = `commerce-sdk@${version}`;
   });
 
   afterEach(nock.cleanAll);
 
   it("identifies commerce-sdk and version", async () => {
-    mockRequestWithUserAgent(sdkUserAgent);
-    const client = new ShopperCustomers({
+    await mockRequestWithUserAgent(sdkUserAgent);
+    const client = new ShopperLogin.ShopperLogin({
       baseUri: "http://somewhere",
       parameters: { organizationId: "foo" },
     });
     await client.authorizeCustomer({
-      body: {},
+      parameters: params,
```

Comment:
Just to clarify this change, we're now using authorizeCustomer from ShopperLogin instead of from ShopperCustomers and it expects a parameters object instead of a request body?